### DISCUSSION TITLE
Skip charging limit

### DISCUSSION THREAD

**Discussion by centuryblack9** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1265] [Q&A] [answered]
Hey guys, I have a charging limit of 70% and it works well. I want to know if there is any way to fully charge my mac without quitting the AlDente. For the next few days, I'll be travelling and working along side so want to keep my MacBook's battery full before I leave my room.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @centuryblack9,

If you have AlDente Pro, you can click the "Top Up" Button. This will result in your MacBook charging to 100% once. If you have AlDente Free, you can manually change the slider to 100% and change it back to 70% when you do not need the full battery anymore.

Best,
Matthias
